okapi-core/src/main/java/org/folio/okapi/managers/ProxyService.java

```diff
@@ -132,62 +133,13 @@ private void makeTraceHeader(ModuleInstance mi, int statusCode,
     pc.logResponse(mi.getModuleDescriptor().getId(), url, statusCode);
   }
 
-  private boolean match(RoutingEntry e, HttpServerRequest req) {
+  private static boolean match(RoutingEntry e, HttpServerRequest req) {
```

Comment:
Looks like this method is not used anymore.

Reply:
Right